mealie/services/scheduler/scheduler_service.py

```diff
@@ -42,14 +42,21 @@ async def schedule_daily():
         hour_target = 23
         minute_target = 45
 
-    hours_until = ((hour_target - now.hour) % 24) or 24
-    minutes_until = (minute_target - now.minute) % 60
-    logger.debug("Hours until %s and minutes until %s", str(hours_until), str(minutes_until))
-
-    delta = timedelta(hours=hours_until, minutes=minutes_until)
-    target_time = (now + delta).replace(microsecond=0, second=0)
+    next_schedule = now.replace(hour=hour_target, minute=minute_target, second=0, microsecond=0)
+    delta = next_schedule - now 
```

Comment:
```suggestion
    delta = next_schedule - now
```

This is what's causing your CI to fail

Reply:
Thank you. My bad, forgot to run formatter.

Reply:
No stress. If you're running in a dev container and following our [recommended workflow with dev containers](https://docs.mealie.io/contributors/developers-guide/starting-dev-server/), it should run automatically for you (as part of the pre commit stage, I think). But no stress if not.

Reply:
Yes, I've missed that for some reason, thank you again